internal/controller/streams.go

```diff
@@ -203,15 +206,48 @@ func (sm *streamManager) StreamCiliumNetworkFlows(ctx context.Context, ciliumNam
 			err = ciliumFlowCollector.exportCiliumFlows(ctx, *sm)
 			if err != nil {
 				sm.logger.Warnw("Failed to collect and export flows from Cilium Hubble Relay", "error", err)
-				sm.streamClient.disableNetworkFlows = true
+				sm.streamClient.disableNetworkFlowsCilium = true
 				return err
 			}
 		}
 	}
 	return nil
 }
 
-// connectAndStreamCiliumNetworkFlows creates ciliumNetworkFlows client and begins the streaming of network flows.
+// StreamFalcoNetworkFlows handles the falco network flow stream.
+func (sm *streamManager) StreamFalcoNetworkFlows(ctx context.Context) error {
+	var falcoFlow string
+	for {
+		falcoFlow = <-sm.streamClient.falcoEventChan
+		if filterIllumioTraffic(falcoFlow) {
+			// Extract the relevant part of the output string
+			re := regexp.MustCompile(`\((.*?)\)`)
+			match := re.FindStringSubmatch(falcoFlow)
+			if len(match) < 2 {
+				return nil
+			}
+
+			podInfo, err := parsePodNetworkInfo(match[1])
+			if err != nil {
+				return nil
```

Comment:
Add a comment, as this is surprising at first glance.
```suggestion
				// If the event can't be parsed, consider that it's not a flow event and just ignore it.
				return nil
```